apps/admin/backend/package.json

```diff
@@ -99,8 +91,7 @@
     "nodemon": "^2.0.20",
     "sort-package-json": "^1.50.0",
     "supertest": "^6.0.1",
-    "ts-jest": "^29.1.1",
-    "typescript": "4.6.3"
```

Comment:
I also moved `typescript` to the workspace root where possible, as it's universal to the repo.